File: RP4/d2.java
package RP4;

import java.util.Scanner;

public class d2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double[] A = new double[20];
        double[] B = new double[20];
        double[] C = new double[20];

        System.out.println("Digite 20 números para A");
        for (int i = 0; i < 20; i++) {
            System.out.print("A[" + i + "]: ");
            A[i] = scanner.nextDouble();
        }

        System.out.println("Digite 20 números para B");
        for (int i = 0; i < 20; i++) {
            System.out.print("B[" + i + "]: ");
            B[i] = scanner.nextDouble();
        }

        for (int i = 0; i < 20; i++) {
            C[i] = A[i] - B[i];
        }

        System.out.println("Resultado da matriz C:");
        for (int i = 0; i < 20; i++) {
            System.out.println(C[i]);
        }

        scanner.close();
    }
}
    

File: RP4/d2.py
A = []
B = []
C = []

print("Digite 20 números para A")

for i in range(20):
    A.append(float(input(f"A[{i}]: ")))

print("Digite 20 números para B")

for i in range(20):
    B.append(float(input(f"B[{i}]: ")))

for i in range(20):
    C.append(A[i] - B[i])

print("Resultado da matriz C:")

for i in range(20):
    print(C[i])